setup/__main__.py

```diff
@@ -161,11 +154,14 @@ async def main(force: bool | None = None) -> None:
                 value,
             )
 
-    if bundle_data := DATA.get("initialBundle"):
-        if not isinstance(bundle_data, dict):
-            logging.warning("Invalid initial bundle data")
-        else:
-            await create_initial_bundle(bundle_data)
+        if bundle_data := template.get("initialBundle"):
+            if not isinstance(bundle_data, dict):
+                logging.warning("Invalid initial bundle data")
+            else:
+                await create_initial_bundle(bundle_data)
```

Comment:
Is it correct that this changed in indentation?

Reply:
yep. it should only happen during the initial setup. so when force_install is false, bundle data will be "None" anyways.